### ISSUE TITLE
Build process cutting color code - Outlook problem.

### ISSUE THREAD

**Issue by Chellios** [foundation/foundation-emails#807]
If I run **npm run build** every color-code that are able to cut into 3 chars, will be cut during the build process. For example the color #999999 will be cut into #999. Also if I write this color directly inline.
But this short color-code isn't known by Outlook. So Outlook displays his default color-code for the element.
The only workaround until now is to modify the output file, but that isn't really beaty, because this file is inlined and minified.
It would be better to modify the build-process once, that every color-code will be write in its original length of 6 characters.

**Comment by davehoran** [comment]
What version of the system are you using?

**Comment by Chellios** [comment]
I'm using the SASS version with Inky.

**Comment by davehoran** [comment]
I checked the code I just generated this morning. I'm seeing the same behavior for colors like #ffffff being converted to #fff. However, I haven't had any issues with outlook in this regard. Have you tested the generated code against several versions and platforms of Outlook to see if there is a pattern? Do you use a tool like Litmus?

**Comment by Chellios** [comment]
Yes I've tested in many versions of Outlook and found the bug in Microsoft Office Professional Plus 2013. But I didn't use Litmus, that I can't use in my case.

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
